spec/unit/pull_request_spec.rb

```diff
@@ -8,7 +8,58 @@
     end
   end
 
-  describe '.to_commit' do
+  describe '#story_ids' do
+    let(:story_id) { random_story_id }
+
+    subject { PullRequest.new(title: title).story_ids }
+
+    context 'no story ID' do
+      let(:title) { Faker::Lorem.sentence }
+
+      it 'returns an empty array' do
+        expect(subject).to be == []
+      end
+    end
+
+    context 'story ID without #' do
+      let(:title) { "[#{story_id}] Faker::Lorem.sentence" }
+
+      it 'returns an empty array' do
+        expect(subject).to be == []
+      end
+    end
+
+    context 'story ID at the beginning' do
+      let(:title) { "[##{story_id}] #{Faker::Lorem.sentence}" }
+
+      it 'returns the story ID' do
+        expect(subject).to be == [story_id.to_s]
+      end
+    end
+
+    context 'multiple story IDs' do
+      let(:story_ids) { (1..3).collect { random_story_id} }
+      let(:title) { "#{ids_string} #{Faker::Lorem.sentence}" }
+
+      context 'separate brackets' do
+        let(:ids_string) { story_ids.collect {|id| "[##{id}]" }.join ' ' }
```

Comment:
Prefer `map` over `collect`.<br>Space between { and | missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.